Dockerfile

```diff
@@ -111,7 +111,8 @@ RUN set -ex; \
     rm -rf ~/.cache/pip; \
     apt-get purge -y --auto-remove $buildDeps
 
-ENV CLICKHOUSE_SERVER clickhouse-server:9000
+# XXX: Why is this defined here?
+ENV CLICKHOUSE_HOST clickhouse-server
```

Comment:
I don't understand why this specific host name is set here and would like to remove it unless somebody knows the reason why it's here.

Reply:
This might be related to the [Travis configuration](https://github.com/getsentry/snuba/blob/master/.travis.yml#L12) — if that's the case, it should just just be set as an environment variable there, rather than defaulting here.